packages/core/src/Badge/Badge.spec.tsx

```diff
@@ -81,4 +81,102 @@ describe('Badge', () => {
     expect(json).toHaveStyleRule('background-color', theme.colors.lightBlue)
     expect(json).toHaveStyleRule('color', theme.colors.text)
   })
+
+  test('badgeStyle primary sets background-color and color', () => {
+    const json = rendererCreateWithTheme(<Badge badgeStyle='primary' />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('background-color', theme.colors.blue)
+    expect(json).toHaveStyleRule('color', theme.colors.lightestText)
+  })
```

Comment:
You could combine these test cases since they all have the same setup with different variables

https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m